specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -2414,7 +2417,27 @@
       "properties": {
         "workLoadType": {
           "description": "Type of workload for the backup management",
-          "type": "string"
+          "enum": [
```

Comment:
not clear from the back and forth on this breaking change.  did your server side already only support these values when it was exposed as a string?